CHANGELOG.md

```diff
@@ -5,11 +5,29 @@ Please follow indidivudal releases for more information.
 
 Note - Graphback is still not ready to be used in production.
 
+## 0.5.0 (1 August, 2019)
+### Graphback
+#### Features
+- Added ability to choose between methods using config and directives.
+- Added ability for custom Queries and Mutations.
+
+#### Fixes
+- Modularized resolver output into types.
```

Comment:
Even for me this sounds vague. Not sure what this means. We need more info at least couple sentences.